ross/shaft_element.py

```diff
@@ -616,24 +616,23 @@ def patch(self, position, SR, ax):
             )
         )
 
-    def bokeh_patch(self, position, SR, check_sld, bk_ax):
+    def bokeh_patch(self, position, check_sld, bk_ax):
```

Comment:
We still have some coupling on the code here (`check_sld`), but I will merge this now and later we can test a way of decoupling this.